src/definitions/behaviors/form.js

```diff
@@ -368,7 +368,13 @@ $.fn.form = function(parameters) {
             if (initialValue == null) { initialValue = ''; }
             var currentValue = $el.val();
             if (currentValue == null) { currentValue = ''; }
-
+            // multiple select values are returned as arrays which are never equal, so do string conversion first
+            if(Array.isArray(currentValue)) {
```

Comment:
I think this `if` branch can be combined with the above `if`, since `currentValue` is not an `Array` if it was `null`.

```suggestion
            else if(Array.isArray(currentValue)) {
```

Reply:
Done by 57ba588f8ef8ba8c78e5c772f697c009767da86b (also did that for `initialValue`)